pkg/cluster/internal/providers/podman/provider.go

```diff
@@ -53,6 +56,10 @@ func (p *Provider) Provision(status *cli.Status, cluster string, cfg *config.Clu
 	if err := ensureMinVersion(); err != nil {
 		return err
 	}
+	// warnf if podman is running rootless
+	if os.Geteuid() != 0 {
+		p.logger.Warn("podman might not work properly in rootless mode")
```

Comment:
maybe something like this:
```suggestion
		p.logger.Warn("kind may not work properly with rootless podman")
```
or this:
```suggestion
		p.logger.Warn("podman provider may not work properly in rootless mode")
```

_podman_ works in rootless, but podman + kind may not